code/modules/atmospherics/components/unary/vent_scrubber.dm

```diff
@@ -77,8 +77,10 @@
 		icon_state = "weld"
 	else if((stat & NOPOWER) || !use_power)
 		icon_state = "off"
+	else if(scrubbing == "exchange")
```

Comment:
While it's true that the current code is nonsensical, there do seem to be defines you should be using for this.

Reply:
Done.

Reply:
If you did, you didn't push the commit.

Reply:
Shit